pandas/io/formats/format.py

```diff
@@ -1721,10 +1721,10 @@ def format_percentiles(
 
     percentiles = 100 * percentiles
 
-    int_idx = np.isclose(percentiles.astype(int), percentiles)
+    int_idx = np.isclose(percentiles.round().astype(int), percentiles)
```

Comment:
Could you make `percentiles.round().astype(int)` a dedicated variable so it can be used here and below?